src/app/api/pdfMonkey/pdfMonkey.api.ts

```diff
@@ -0,0 +1,70 @@
+import { PDFMONKEY_DOCUMENT_ID } from "@/models/constants";
+import { Booking, Boat } from "@/models/models";
+import moment from "moment";
+
+const pdfMonkey_api_key = process.env.PDFMONKEY_API_KEY
```

Comment:
Remember to add tot he readme and to the env.default

Metnion on the description so we added to netlify before we release